packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -49,13 +49,15 @@ class PinCodeBloc extends $PinCodeBloc {
     required this.biometricAuthenticationService,
     required this.pinCodeService,
     required this.localizedReason,
+    required this.autoBiometricAuth,
   }) {
     authenticated.connect().addTo(_compositeSubscription);
   }
 
   final PinBiometricsService biometricAuthenticationService;
   final PinCodeService pinCodeService;
   final String localizedReason;
+  final bool autoBiometricAuth;
```

Comment:
Do you think the this name fits better`autoPromptBiometric` ?

Reply:
Yes, that does sound better